framework/packages/abstract-client/Cargo.toml

```diff
@@ -14,7 +14,7 @@ readme = "README.md"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-default = ["test-utils"]
+default = ["test-utils", "interchain"]
```

Comment:
Are we sure we want to enable this by default? It makes this a huge dependency by default if we do.

Reply:
> Are we sure we want to enable this by default? It makes this a huge dependency by default if we do.

Agree, let's not enable it by default